lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -1003,3 +1015,6 @@ keyword(_) -> false.
 
 atom_length(Atom) ->
   length(atom_to_list(Atom)).
+
+invalid_character_error(Char) ->
+  "invalid character '" ++ [Char] ++ "' in name: ".
```

Comment:
I think we should rather go with "unexpected token: ", which we already show in other parts of the grammar. :)